lib/commands.js

```diff
@@ -467,9 +467,18 @@ let commands = {
 
     // remove any modifier keys from the inputKeys array and put them in a new array
     inputKeys.forEach((key) => {
-      // change command to control on windows
-      if (key === "command" || key === "control") {
-        key = commandOrControl;
+      // on Windows: command should map to command (Windows key), control should map to control
+      // on Mac: both command and control remain as-is
+      if (platform() === "win32") {
+        // on Windows, keep command as command (robotjs treats this as Windows key)
+        // and keep control as control
+        if (key === "control") {
+          key = "control";
```

Comment:
```suggestion
      // on Windows: existing "win" modifier key above should map to command (Windows key), control should map to control 
      // on Mac: both command and control remain as-is
      if (platform() === "win32") {
        // on Windows, keep command as command (robotjs treats this as Windows key)
        // and keep control as control
        if (key === "win") {
          key = "command";
```
otherwise we would map every instance of cmd (mac) > cmd (windows) when this is just a special case. We have the "win" modifier key in the array above, let's just use that so it's not ambiguous